### DISCUSSION TITLE
I'm a Beginner

### DISCUSSION THREAD

**Discussion by gksolutions33** [microsoft/Web-Dev-For-Beginners#534] [Q&A]
I'm a Beginner where to start

**Comment by BigBro17on25tolife** [comment]
Same here, I went and purchased books on Java and python; i’m thinking only way to keep up, is to start at begging. It’s like everything, difficult at first but you’ll start to overstand it. Keep going don’t quit.

**Comment by jlooper** [comment]
If you go through the lessons here from the beginning it will give you a good intro to web dev. Follow the home README's table for the lessons

**Comment by Richlord789** [comment]
I am a beginner

**Reply by Huzaifwani88** [reply]
Fresher what is the basic structure of Github 
I don't know where to start

**Reply by jlooper** [reply]
Please start by reading the base README in this repo and then start the lessons from the table

**Reply by GoodnessEzeh** [reply]
Good advice. Going through the ReadMe file would be a good start. It's pretty easy to understand.

**Reply by PEminent** [reply]
It's a great intro to get insight into the world of web

**Comment by NOISE440J** [comment]
I'm very enthusiastic to star this journey !!!

**Reply by PEminent** [reply]
This is a nice resources to get started

**Reply by jlooper** [reply]
:) thanks!

**Comment by mannytee1** [comment]
Hi I just Joined and very eager to learn as well, how do i go about her on Github pls. Started with web dev...html. thanks

**Reply by jlooper** [reply]
hi, follow the lessons from the start; GitHub comes up pretty quickly in the introduction to get you started
